__main__.py

```diff
@@ -6,31 +6,35 @@
 from message_adapter import message_adapter
 
 if __name__ == '__main__':
-  functionName = sys.argv[1]
-  transformer = message_adapter.message_adapter()
-  exitCode = 1
-  allInput = json.loads(raw_input())
-  event = allInput['event']
-  try:
-    if (functionName == 'loadRemoteEvent'):
-      result = transformer.loadRemoteEvent(event)
-    elif (functionName == 'loadNestedEvent'):
-      context = allInput['context']
-      result = transformer.loadNestedEvent(event, context)
-    elif (functionName == 'createNextEvent'):
-      handlerResponse = allInput['handler_response']
-      if 'message_config' in allInput:
-        messageConfig = allInput['message_config']
-      else:
-        messageConfig = None
-      result = transformer.createNextEvent(handlerResponse, event, messageConfig)
+    functionName = sys.argv[1]
+    transformer = message_adapter.message_adapter()
```

Comment:
transformer = message_adapter.message_adapter()
This could take the schemas from the input message if exists.

Reply:
:+1: Thanks I agree, that seems like a good idea.